Data/DataModel/ContentFile.cs

```diff
@@ -11,6 +11,19 @@ namespace ServerCore.DataModel
     /// </summary>
     public class ContentFile
     {
+        public ContentFile()
+        {
+        }
+
+        public ContentFile(ContentFile source)
+        {
+            FileType = source.FileType;
+            Event = source.Event;
+            ShortName = source.ShortName;
+            UrlString = source.UrlString;
```

Comment:
Would it make sense to exclude the properties that should be rewritten? The only persistent properties are FileType and ShortName

Reply:
I'm trying to have every class copy everything. I agree it's overkill in this case but it will make it easier to find missing properties overall.